src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -185,16 +187,32 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
               })}
             <Projects orgId={organization.slug} slugs={[this.projectId]}>
               {({projects}) => (
-                <BorderlessEventEntries
-                  api={api}
-                  organization={organization}
-                  event={event}
-                  project={projects[0]}
-                  location={location}
-                  showExampleCommit={false}
-                  showTagSummary={false}
-                  eventView={eventView}
-                />
+                <SpanEntryContext.Provider
+                  value={{
+                    getViewChildTransactionTarget: childTransactionProps => {
+                      const childTransactionLink = eventDetailsRoute({
+                        eventSlug: childTransactionProps.eventSlug,
+                        orgSlug: organization.slug,
+                      });
+
+                      return {
+                        pathname: childTransactionLink,
+                        query: eventView.generateQueryStringObject(),
+                      };
+                    },
+                  }}
+                >
+                  <BorderlessEventEntries
+                    api={api}
+                    organization={organization}
+                    event={event}
+                    project={projects[0]}
+                    location={location}
+                    showExampleCommit={false}
+                    showTagSummary={false}
+                    eventView={eventView}
```

Comment:
If we generated all the traversal links via the context, would we need eventView as a prop?

Reply:
I've removed the eventview prop drilling in https://github.com/getsentry/sentry/pull/19410/commits/d038d48c20d95c8823faf516b763da1d585b6db6 and https://github.com/getsentry/sentry/pull/19410/commits/a8632936648a393ab782867b3b84ee2c92fcfdc3